packages/visual-editor/src/components/RootRenderer/RootRenderer.tsx

```diff
@@ -32,11 +34,27 @@ export const RootRenderer: React.FC<Props> = ({ onChange }) => {
     if (onChange) onChange(tree);
   }, [tree, onChange]);
 
+  const wrapperRef = useRef<HTMLDivElement>(null);
```

Comment:
Whats wrapperRef doing? It seems likes its only used to check if the ref exists, but it seems like the div its attached to will always be there as its not conditionally rendered.